http/thumbnail.go

```diff
@@ -0,0 +1,92 @@
+package http
+
+import (
+	"github.com/disintegration/imaging"
+	"github.com/filebrowser/filebrowser/v2/errors"
+	"github.com/filebrowser/filebrowser/v2/files"
+	"image"
+	"image/gif"
+	"image/jpeg"
+	"image/png"
+	"mime"
+	"net/http"
+	"net/url"
+)
+
+var thumbnailHandler = withUser(func(w http.ResponseWriter, r *http.Request, d *data) (int, error) {
+	if !d.user.Perm.Download {
+		return http.StatusAccepted, nil
+	}
+
+	file, err := files.NewFileInfo(files.FileOptions{
+		Fs:      d.user.Fs,
+		Path:    r.URL.Path,
+		Modify:  d.user.Perm.Modify,
+		Expand:  true,
+		Checker: d,
+	})
+	if err != nil {
+		return errToStatus(err), err
+	}
+
+	if file.IsDir || file.Type != "image" {
+		return http.StatusNotFound, nil
```

Comment:
How about returning StatusNotImplemented code?
```suggestion
		return http.StatusNotImplemented, nil
```

Reply:
ok.